Makefile

```diff
@@ -169,13 +169,13 @@ endif
 .PHONY: docker-image
 docker-image:
 	docker pull ghcr.io/microsoft-cisl/mlos/mlos-build-ubuntu-$(UbuntuVersion):latest
-	docker build . --target $(MlosBuildImageTarget) \
+	docker build . $(DOCKER_BUILD_ARGS) --target $(MlosBuildImageTarget) \
```

Comment:
A convenience variable to allow me to pass `--pull` to the `docker build` command for forced refresh of the underlying local image caches.